src/test/java/app/library/management/core/repository/file/FileStorageTest.java

```diff
@@ -0,0 +1,108 @@
+package app.library.management.core.repository.file;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileStorageTest {
+
+    private Path tempFilePath;
+    private FileStorage fileStorage;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        tempFilePath = Files.createTempFile("testFileStorage", ".json");
+        String initString = "[]";
+        Files.write(tempFilePath, initString.getBytes());
+        fileStorage = new FileStorage(tempFilePath.toString());
+    }
+
+    @AfterEach
+    public void tearDown() throws IOException {
+        Files.deleteIfExists(tempFilePath);
+    }
```

Comment:
```suggestion
public abstract class FileStorageSupport {

    protected Path tempFilePath;
    protected FileStorage fileStorage;

    @BeforeEach
    public void setUp() throws IOException {
        tempFilePath = Files.createTempFile("testFileStorage", ".json");
        String initString = "[]";
        Files.write(tempFilePath, initString.getBytes());
        fileStorage = new FileStorage(tempFilePath.toString());
    }

    @AfterEach
    public void tearDown() throws IOException {
        Files.deleteIfExists(tempFilePath);
    }
}
```

설정이 많아지면 Support 클래스로 분리할 수도 있을 것 같습니다.